javascript/javascript1/week3/itemremoval.js

```diff
@@ -0,0 +1,21 @@
+const names = [
+  "Peter",
+  "Ahmad",
+  "Yana",
+  "kristina",
+  "Rasmus",
+  "Samuel",
+  "katrine",
+  "Tala",
+];
+const nameToRemove = "Tala"; //This is adjustable to see if console.logs will be different
+
+// Write some code here
+for (i = 0; i < names.length; i++) {
+  if (names[i] === nameToRemove) {
+    names.splice(i, 1);
```

Comment:
Once the name is found, you can stop the for loop so it does not spend time looking up the rest of the array.
Imagine if the name to find was, for example, in the 2nd or 3rd position in the array.